workers/shareScrollWorker.js

```diff
@@ -113,43 +115,57 @@ async function getBannerBaseComposite(input) {
         left: 118,
     });
     // flair
+    // console.log('CHOSEN FLAIR: ', input.user.flairPath); // flair debugging
     if (input.user.flairPath) {
-        const flairPath = path.resolve('/src/resources/public', input.user.flairPath);
-        const flairShadow = sharp(flairPath)
-            .greyscale()
-            .threshold(255)
-            .composite([
-            {
-                input: Buffer.from([255, 255, 255, Math.ceil(255 / 5)]),
-                raw: {
-                    width: 1,
-                    height: 1,
-                    channels: 4,
+        const shadowPath = path.resolve('/src/resources/public/', input.user.flairPath.replace('items', 'items/shadows'));
```

Comment:
Be nice if we could move this from public. No real reason for people to want to access them